_data/compatibility/coinbase.yaml

```diff
@@ -0,0 +1,84 @@
+---
+name: Coinbase
+internal_url: /en/compatibility/coinbase
+logo: /img/compatibility/coinbase/coinbase.png
+rbf:
+  tested:
+    date: "2018-11-05"
+    platforms:
+      - web
+    version: "n/a"
+  features:
+    receive:
+      notification: "false"
+      list: "false"
+      details: "false"
+      shows_replaced_version: "false"
+      shows_original_version: "true"
+    send:
+      signals_bip125: "false"
+      list: "untested"
+      details: "untested"
+      shows_replaced_version: "untested"
+      shows_original_version: "untested"
+  examples:
+    - image: /img/compatibility/coinbase/rbf/send-screen-default.png
+      caption: >
+        Sending RBF Transaction - Default send transaction screen.
+    - image: /img/compatibility/coinbase/rbf/send-confirm.png
+      caption: >
+        Sending RBF Transaction - Send transaction confirmation screen. Shows fees. No RBF flag. Transaction sent without RBF signaled.
+    - image: /img/compatibility/coinbase/rbf/transaction-details-sent.png
+      caption: >
+        Bumping RBF Transaction - Transaction not sent with RBF so no bumping possible.
+    - image: /img/compatibility/coinbase/rbf/transaction-details-sent-2.png
+      caption: >
+        Bumping RBF Transaction - After a period of time the View Transaction link shows up.
+    - image: /img/compatibility/coinbase/rbf/transaction-list-incoming-rbf.png
+      caption: >
+        Receiving RBF Transaction - Incoming RBF transaction list. No RBF label.
+    - image: /img/compatibility/coinbase/rbf/transaction-details-incoming-rbf.png
+      caption: >
+        Receiving RBF Transaction - Incoming RBF transaction details. No RBF label. Further transaction details are at BlockCypher explorer which does not label RBF transactions.
+    - image: /img/compatibility/coinbase/rbf/transaction-list-incoming-replacement.png
+      caption: >
+        Receiving Bumped RBF Transaction - After bumped transaction confirmed, the bumped transaction then shows up and is credited. Original transactions stays as pending (even after 100 confirmations).
```

Comment:
Nit: stays -> stay to agree with plural txs.

Reply:
good catch. corrected.